docs/features.md

```diff
@@ -4,7 +4,6 @@
 |---------|-----------|---------------------------|-----------------------------------|--------|
```

Comment:
At `docs/development-and-gardener-setup.md` there is still a mentioning of OPA policies which should be deleted as well.
```
(plus some additional configuration of demo OPA policies)
```

Reply:
Done.